pkg/proxy/iptables/proxier.go

```diff
@@ -552,6 +572,61 @@ func servicePortEndpointChainName(s proxy.ServicePortName, protocol string, endp
 	return utiliptables.Chain("KUBE-SEP-" + encoded[:16])
 }
 
+// getRemovedEndpointIPs returns the endpoint IPs that are missing in the new endpoints
+func getRemovedEndpoints(curEndpoints, newEndpoints []string) []string {
+	var result []string
+	epSet := sets.NewString(newEndpoints...)
+	for _, ep := range curEndpoints {
+		if epSet.Has(ep) {
+			result = append(result, ep)
+		}
+	}
+	return result
+}
+
+type endpointServicePair struct {
+	endpoint        string
+	servicePortName proxy.ServicePortName
+}
+
+// deleteEndpointConnection deletes all udp service connections to specified endpoint in the map to avoid stale endpoints black hole traffic
```

Comment:
This comment should be clearer.  Someting like 

```
// After a UDP endpoint has been removed, we must flush any pending conntrack entries to it, or else we
// risk sending more traffic to it, all of which will be lost (because UDP).
```

Maybe clarify the comment on the next func, too.